seed/compileSeedData/index.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const S3PATH = 'https://product-carousel-images.s3-us-west-2.amazonaws.com/';
+
+const imageNameString = fs
+  .readFileSync(path.resolve(__dirname, '..', 'retrieveImgs', 'images.txt'))
+  .toString();
+
+const imageNames = imageNameString.split('\n');
+
+const seedJson = [];
+
+imageNames.forEach((imageName) => {
+  const isThumnail = imageName.includes('thumbnail');
```

Comment:
isThum"b"nail? Totally understand if this intended just a little awkward to read.

Reply:
Ah, good eye! Will fix this.